src/main/components/RequestBar.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useState } from 'react';
 
-const UrlBar = (props) => {
-  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+const RequestBar = (props) => {
+  const method = (props.SourceOrDest === 'dest' ? 'POST' : 'GET')
   const [selected, setSelected] = useState(method);
 
-  function sendFetch () {
+  function sendFetch (e) {
+    e.preventDefault();
     const sendingObj = { method: selected, mode: 'cors' };
     if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
 
-    // console.log('hello;', document.querySelector('#urlInput').value);
-
     fetch(document.querySelector('#urlInput').value, sendingObj)
-      //.then(r => r.json())
-      .then(r1 => console.log(r1));
+      .then(r => console.log(r));
```

Comment:
with using the fetch function, simply returns an HTTP response and not the actual json. To extract the JSON body content from the response, we use the json() method in the first resolution of the promise.  This should help [fetchAPI](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)